ZelBack/src/services/fluxService.js

```diff
@@ -798,7 +1014,7 @@ module.exports = {
   fluxWarnLog,
   fluxInfoLog,
   fluxDebugLog,
-  adjustCruxID,
+  // adjustCruxID, // Deprecated.
```

Comment:
Still keep for now as function is referenced in api. Soon will be most likely changed to FIO